src/steps/video-setup/index.tsx

```diff
@@ -85,7 +85,7 @@ export const VideoSetup: React.FC<StepProps> = ({ goToNextStep }) => {
           title={t("sources-video-user-selected")}
           {...showButtons && buttons}
         >
-          <SourcePreview inputs={[userInput]} />
+          <SourcePreview inputs={[userInput]} isHighContrast />
```

Comment:
You are unconditionally passing `isHighContrast={true}` here, right? That's surely not intended? Also, `SourcePreview` can easily call `useColorScheme` as well, so no need to pass it as prop.